sql/ai.control

```diff
@@ -1,6 +1,6 @@
 # pgai extension
 comment='helper functions for ai workflows'
-default_version='0.3.0'
+default_version='0.4.0'
```

Comment:
If you're worried about having the version in multiple files, we can set up something like release-please to take care of it. We use it in other repos and it works really well.

Reply:
yeah, i want to at least make sure the the version is correct in all the places, but having it automatically filled out would be nice

Reply:
fixed